components/dashboard/src/settings/Account.tsx

```diff
@@ -47,7 +47,7 @@ export default function Account() {
 
         <PageWithSubMenu subMenu={settingsMenu}  title='Account' subtitle='Manage account and git configuration.'>
             <h3>Profile</h3>
-            <p className="text-base text-gray-500 pb-4 max-w-2xl">The following information will be used to set up git configuration. You can override git author name and email per project by using the default environment variables <CodeText>GIT_AUTHOR_NAME</CodeText> and <CodeText>GIT_COMMITTER_EMAIL</CodeText>.</p>
+            <p className="text-base text-gray-500 pb-4 max-w-2xl">The following information will be used to set up git configuration. You can override git author name and email per project by using the default environment variables <CodeText>GIT_AUTHOR_NAME</CodeText>, <CodeText>GIT_COMMITTER_NAME</CodeText>, <CodeText>GIT_AUTHOR_EMAIL</CodeText> and <CodeText>GIT_COMMITTER_EMAIL</CodeText>.</p>
```

Comment:
I think it might make sense to refer directly to the git docs, rather than try to copy too much of the info here. 

It seems by adding the detail here about `GIT_AUTHOR_NAME` and `GIT_COMMITTER_EMAIL` we already created some confusion (#6028), so I'd suggest we instead directly reference the git docs. 

For example: 

1. We're missing the context from the docs regarding the fact that these are only used when global config is not already set. 

> uses these environment variables as its primary source of information, falling back to configuration values only if these aren’t present.

2. We're also missing some other environment variable configurations like `GIT_COMMITTER_DATE`, etc. 

So I'd suggest...

```suggestion
            <p className="text-base text-gray-500 pb-4 max-w-2xl">The following information will be used to set up your workspace git configuration. If required, you can override these values using <a target="_blank" href="https://git-scm.com/book/en/v2/Git-Internals-Environment-Variables#_committing">environment variables</a>. </p>
```